systest/backup/minio/backup_test.go

```diff
@@ -117,7 +117,7 @@ func TestBackupMinio(t *testing.T) {
 	dirSetup(t)
 
 	// Send backup request.
-	// TODO: minio backup request fails when the environment is not ready, 
+	// TODO: minio backup request fails when the environment is not ready,
```

Comment:
@SashaDgraph I guess this was missed in your checks, but it's okay (an extra space was cleaned up ~ possibly by your IDE)